includes/transformer/class-post.php

```diff
@@ -894,11 +894,15 @@ protected function get_post_content_template() {
 		$template = $content ?: ACTIVITYPUB_CUSTOM_POST_CONTENT; // phpcs:ignore Universal.Operators.DisallowShortTernary.Found
 
 		$post_format_setting = \get_option( 'activitypub_object_type', ACTIVITYPUB_DEFAULT_OBJECT_TYPE );
+		$type                = $this->get_type();
 
 		if ( 'wordpress-post-format' === $post_format_setting ) {
 			$template = '';
 
-			if ( 'Note' === $this->get_type() ) {
+			// If the post is a note and not a reply, force the inclusion of the post title.
+			$is_reply = null !== $this->get_in_reply_to();
+
```

Comment:
Should we also check for `$this->get_mentions()`?

Reply:
I worry this would catch longer posts, that would only happen to mention Fediverse users like we mention someone by their name and link to their blog or Twitter account today. 

Here is an example:
https://herve.bzh/where-did-all-the-wordpress-editors-go/